web-api/src/business/useCases/document/batchDownloadDocketEntriesInteractor.ts

```diff
@@ -93,7 +104,7 @@ const batchDownloadDocketEntriesHelper = async (
   const { caseCaption, isSealed: isCaseSealed } = caseToBatch;
   const caseTitle = Case.getCaseTitle(caseCaption);
   const caseFolder = `${docketNumber}, ${caseTitle}`;
-  const zipName = `${caseFolder}.zip`;
+  const zipName = `${uuid}-${batchIndex} ${caseFolder}.zip`;
```

Comment:
Is it better to avoid spaces in a file name?

Reply:
I think the file name already has spaces in it, 
`const caseFolder = `${docketNumber}, ${caseTitle}`; `
But I think I might remove caseFolder from this batch zip name (keep names short and using uuid guarantees unique name).